src/Symfony/Contracts/Translation/.gitattributes

```diff
@@ -0,0 +1,3 @@
+/Test export-ignore
```

Comment:
Should be reverted. The `Test` namespace contains files that people can reuse in their own tests. The `Tests` namespace contains tests that we can indeed ignore.